.circleci/config.yml

```
@@ -189,7 +189,10 @@ jobs:
           name: Run Docker build
           working_directory: ~/project/solidity/dashboard
           command: |
-            docker build --tag keep-dapp-token-dashboard .
+            docker build \
+            --build-arg ETH_RPC_URL=$ETH_RPC_URL \
+            --build-arg CHAIN_ID=$ETH_NETWORK_ID \
```

Thoughts about renaming `CHAIN_ID` to `ETH_NETWORK_ID`?

```suggestion
            --build-arg ETH_NETWORK_ID=$ETH_NETWORK_ID \
```

---

It is related to the [eip-155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md). `networkId` is not used at all in the signing process (the `chainId` is used for signing). That's why I used `chainId` instead of `networkId`. But in most cases especially mainet(`chainId=1` `networkId = 1`) and public test networks eg. ropsten (`chainId = 3` `networkId = 3`) `networkId == chainId`. Maybe we should also add the `ETH_CHAIN_ID` var to circleCI context to avoid misunderstanding?